### ISSUE TITLE
cloud_to_cloud:node:... very flaky (TIMEOUT)

### ISSUE THREAD

**Issue by dgquintas** [grpc/grpc#13253]
https://sponge.corp.google.com/target?id=80ee7ec6-5b64-4ed8-9704-e376ac09e159&target=github/grpc&searchFor=&show=ALL&sortBy=STATUS

For example:
```
cloud_to_cloud:node:csharpcoreclr_server:large_unary 
cloud_to_cloud:node:python_server:large_unary 
cloud_to_cloud:node:ruby_server:client_streaming 
```

**Comment by adelez** [comment]
This dashboard shows how severe the issue is: https://performanceedit-dot-grpc-testing.appspot.com/explore?dashboard=5711255578869760: Node client is the most flaky one.

**Comment by jtattermusch** [comment]
A bit more analysis shows that the node client is behind virtually all cloud_to_cloud interop test failures in the last days. Most failures are `large_unary`, followed by `client_streaming`.
```

grpc/ubuntu/master/grpc_interop_tocloud | 88 | null | cloud_to_cloud:node:c++_server:custom_metadata | cloud_to_cloud | node | c++_server | custom_metadata | TIMEOUT | 2017-11-03 22:05:43 UTC | 181.35 |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2 | grpc/ubuntu/master/grpc_interop_tocloud | 87 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-03 17:48:06 UTC | 186.91 |  
3 | grpc/ubuntu/master/grpc_interop_tocloud | 86 | null | cloud_to_cloud:node:csharpcoreclr_server:large_unary | cloud_to_cloud | node | csharpcoreclr_server | large_unary | TIMEOUT | 2017-11-03 16:28:36 UTC | 187.81 |  
4 | grpc/ubuntu/master/grpc_interop_tocloud | 86 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-03 16:28:43 UTC | 187.56 |  
5 | grpc/ubuntu/master/grpc_interop_tocloud | 85 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-03 15:48:23 UTC | 185.57 |  
6 | grpc/ubuntu/master/grpc_interop_tocloud | 85 | null | cloud_to_cloud:node:python_server:large_unary | cloud_to_cloud | node | python_server | large_unary | TIMEOUT | 2017-11-03 15:47:48 UTC | 187.59 |  
7 | grpc/ubuntu/master/grpc_interop_tocloud | 85 | null | cloud_to_cloud:node:csharp_server:large_unary | cloud_to_cloud | node | csharp_server | large_unary | TIMEOUT | 2017-11-03 15:47:31 UTC | 180.36 |  
8 | grpc/ubuntu/master/grpc_interop_tocloud | 85 | null | cloud_to_cloud:node:python_server:client_streaming | cloud_to_cloud | node | python_server | client_streaming | TIMEOUT | 2017-11-03 15:48:30 UTC | 187.25 |  
9 | grpc/ubuntu/master/grpc_interop_tocloud | 84 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-03 14:46:00 UTC | 181.13 |  
10 | grpc/ubuntu/master/grpc_interop_tocloud | 83 | null | cloud_to_cloud:node:ruby_server:large_unary | cloud_to_cloud | node | ruby_server | large_unary | TIMEOUT | 2017-11-03 09:45:30 UTC | 180.71 |  
11 | grpc/ubuntu/master/grpc_interop_tocloud | 82 | null | cloud_to_cloud:node:python_server:large_unary | cloud_to_cloud | node | python_server | large_unary | TIMEOUT | 2017-11-03 07:46:03 UTC | 188.04 |  
12 | grpc/ubuntu/master/grpc_interop_tocloud | 81 | null | cloud_to_cloud:node:python_server:client_streaming | cloud_to_cloud | node | python_server | client_streaming | TIMEOUT | 2017-11-03 02:23:04 UTC | 181.55 |  
13 | grpc/ubuntu/master/grpc_interop_tocloud | 80 | null | cloud_to_cloud:node:c++_server:client_streaming | cloud_to_cloud | node | c++_server | client_streaming | TIMEOUT | 2017-11-02 15:07:50 UTC | 186.11 |  
14 | grpc/ubuntu/master/grpc_interop_tocloud | 79 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-02 13:31:24 UTC | 182.7 |  
15 | grpc/ubuntu/master/grpc_interop_tocloud | 79 | null | cloud_to_cloud:node:ruby_server:large_unary | cloud_to_cloud | node | ruby_server | large_unary | TIMEOUT | 2017-11-02 13:31:08 UTC | 180.46 |  
16 | grpc/ubuntu/master/grpc_interop_tocloud | 77 | null | cloud_to_cloud:node:python_server:client_streaming | cloud_to_cloud | node | python_server | client_streaming | TIMEOUT | 2017-11-02 03:11:00 UTC | 182.57 |  
17 | grpc/ubuntu/master/grpc_interop_tocloud | 76 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-01 20:54:49 UTC | 183.8 |  
18 | grpc/ubuntu/master/grpc_interop_tocloud | 75 | null | cloud_to_cloud:node:python_server:large_unary | cloud_to_cloud | node | python_server | large_unary | TIMEOUT | 2017-11-01 15:54:38 UTC | 186.2 |  
19 | grpc/ubuntu/master/grpc_interop_tocloud | 75 | null | cloud_to_cloud:node:ruby_server:client_streaming | cloud_to_cloud | node | ruby_server | client_streaming | TIMEOUT | 2017-11-01 15:55:50 UTC | 188.49 |  
20 | grpc/ubuntu/master/grpc_interop_tocloud | 75 | null | cloud_to_cloud:node:csharpcoreclr_server:large_unary | cloud_to_cloud | node | csharpcoreclr_server | large_unary | TIMEOUT | 2017-11-01 15:55:08 UTC | 187.81 |  
21 | grpc/ubuntu/master/grpc_interop_tocloud | 74 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-11-01 13:34:11 UTC | 189.75 |  
22 | grpc/ubuntu/master/grpc_interop_tocloud | 71 | null | cloud_to_cloud:node:c++_server:large_unary | cloud_to_cloud | node | c++_server | large_unary | TIMEOUT | 2017-10-31 22:31:47 UTC | 180.1 |  
23 | grpc/ubuntu/master/grpc_interop_tocloud | 70 | null | cloud_to_cloud:node:csharpcoreclr_server:large_unary | cloud_to_cloud | node | csharpcoreclr_server | large_unary | TIMEOUT | 2017-10-31 16:35:16 UTC | 183.01 |  
24 | grpc/ubuntu/master/grpc_interop_tocloud | 70 | null | cloud_to_cloud:node:csharp_server:client_streaming | cloud_to_cloud | node | csharp_server | client_streaming | TIMEOUT | 2017-10-31 16:35:11 UTC | 188.27 |  
25 | grpc/ubuntu/master/grpc_interop_tocloud | 70 | null | cloud_to_cloud:node:python_server:large_unary | cloud_to_cloud | node | python_server | large_unary | TIMEOUT | 2017-10-31 16:35:00 UTC | 187.33 |  
26 | grpc/ubuntu/master/grpc_interop_tocloud | 70 | null | cloud_to_cloud:node:csharpcoreclr_server:client_streaming | cloud_to_cloud | node | csharpcoreclr_server | client_streaming | TIMEOUT | 2017-10-31 16:35:21 UTC | 182.79 |  
27 | grpc/ubuntu/master/grpc_interop_tocloud | 69 | null | cloud_to_cloud:node:csharp_server:large_unary | cloud_to_cloud | node | csharp_server | large_unary | TIMEOUT | 2017-10-31 13:34:46 UTC | 183.74 |  
28 | grpc/ubuntu/master/grpc_interop_tocloud | 67 | null | cloud_to_cloud:node:ruby_server:client_streaming | cloud_to_cloud | node | ruby_server | client_streaming | TIMEOUT | 2017-10-30 22:35:24 UTC | 186.33 |  
```

the query
```
select * from jenkins_test_results.interop_results where test_name like '%' and job_name like '%grpc_interop_tocloud%' and timestamp > '2017-10-25 00:00' and result != 'PASSED'  order by build_id desc limit 10000 
```

**Comment by dgquintas** [comment]
Ping. This happens consistently.

**Comment by dgquintas** [comment]
Ping. Any update?

**Comment by jtattermusch** [comment]
Another issue that started affecting node (looks like all node interop tests are failing).
```
Failure
/var/local/git/grpc-node/test/any_grpc.js:9
    throw new Error([
    ^

Error: Invalid value for global._client_implementation: undefined. If running from the command line, please --require a fixture first.
    at getImplementation (/var/local/git/grpc-node/test/any_grpc.js:9:11)
    at Object.<anonymous> (/var/local/git/grpc-node/test/any_grpc.js:23:20)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/local/git/grpc-node/test/interop/interop_client.js:23:12)
```

**Comment by jtattermusch** [comment]
That issue has been already filed as https://github.com/grpc/grpc-node/issues/89

**Comment by dgquintas** [comment]
@nicolasnoble and I talked offline about this yesterday. The plan is to have these be disabled while being investigated. @nicolasnoble can you confirm?

**Comment by murgatroid99** [comment]
The fix for that latest issue is #13394.

**Comment by murgatroid99** [comment]
This should be fixed by grpc/grpc-node#95.
